ffmpeg-plugin/configure-ffmpeg.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2024 Intel Corporation
+
+set -e
```

Comment:
Please make this correction as you use `rm FFmpeg -rf` and this may lead to user data being wiped-out. 

```suggestion
set -e
SCRIPT_DIR="$(readlink -f "$(dirname -- "${BASH_SOURCE[0]}")")"
```

Reply:
Let's improve the script in a separate PR. I suggest not to amend this one.